### PR TITLE
scripts: update `drtprod` create drt-large

### PR DESCRIPTION
**Pull request by herkolategan**
Bring the creation in line with what is currently running on cockroach-drt. Uses zone expansion to balance the nodes evenly across regions.

Epic: None
Release Note: None

### CODE REVIEW

File: scripts/drtprod [hunk 1]
```diff
@@ -128,17 +128,21 @@ EOF"
     fi
     case "${2}" in
       "drt-main")
```
Comment (itsbilal) [inline scripts/drtprod:130]: nit: Change this to `drt-large` too?

Comment (herkolategan) [reply scripts/drtprod:130]: yep, was about to double check this before moving it out of draft, thanks for pointing it out!

File: scripts/drtprod [hunk 2]
```diff
@@ -193,22 +193,26 @@ EOF"
...
     case "${2}" in
-      "drt-main")
-        roachprod create drt-main \
+      "drt-large")
+        roachprod create drt-large \
           --clouds gce \
           --gce-managed \
-          --gce-zones "us-east1-b,us-west1-b,europe-west2-b" \
+          --gce-enable-multiple-stores \
+          --gce-zones "northamerica-northeast2-a:2,northamerica-northeast2-b:2,northamerica-northeast2-c:1,europe-central2-a:2,europe-central2-b:2,europe-central2-c:1,northamerica-northeast1-a:2,northamerica-northeast1-b:2,northamerica-northeast1-c:1" \
```
Comment (itsbilal) [inline scripts/drtprod:205]: should we swap Warsaw for Columbus?

**Review by itsbilal — APPROVED** [review]
LGTM

**Review by ajstorm — APPROVED** [review]
LGTM too

File: scripts/drtprod (hunk 2 above)
Comment (herkolategan) [reply scripts/drtprod:237]: Yes, I will update it before merging.

**Comment by herkolategan** [comment]
TFTRs!

bors r=itsbilal,ajstorm

**Pull request merged**